### PR TITLE
Fix HopscotchDemo

### PR DESCRIPTION
**Pull request by Outcue**
Fix title and add missing isMyKind method.

### CODE REVIEW

File: HopscotchDemo.ns [hunk 1]
```diff
@@ -79,10 +79,15 @@ slowFib = (
...
 )
-public class DemoSubject new = Subject onModel: nil (| stopwatch = Stopwatch new. |stopwatch start) (
+public class DemoSubject new = Subject onModel: nil (
+    | stopwatch = Stopwatch new. |
+    stopwatch start) (
 public createPresenter = (
 	^DemoPresenter onSubject: self
 )
+isMyKind: other ^ <Boolean> = (
+    ^other isKindOfDemoPresenter
```
Comment (gbracha) [inline HopscotchDemo.ns:89]: er, shouldn't his be ^other isKindOfDemoSubject

File: HopscotchDemo.ns [hunk 2]
```diff
@@ -79,10 +79,15 @@ slowFib = (
...
 ) : (
 )
-public class DemoSubject new = Subject onModel: nil (| stopwatch = Stopwatch new. |stopwatch start) (
+public class DemoSubject new = Subject onModel: nil (
+    | stopwatch = Stopwatch new. |
+    stopwatch start) (
 public createPresenter = (
 	^DemoPresenter onSubject: self
 )
+isMyKind: other ^ <Boolean> = (
```
Comment (gbracha) [inline HopscotchDemo.ns:88]: I would expect this class to implement isKindOfDemoSubject = (^true)

File: HopscotchDemo.ns [hunk 3]
```diff
@@ -21,7 +21,7 @@ public definition = (
 	^(column: {
 		row: {
 			filler.
-			(label: 'Hello, DOM New World!') color: (Color r: 1 g: 0 b: 0).
+			(label: 'Hello, Hopscotch Demo!') color: (Color r: 1 g: 0 b: 0).
```
Comment (gbracha) [inline HopscotchDemo.ns:24]: DemoPresneter needs to have its own isMyKind:

**Comment by gbracha** [comment]
zwell,it's an improvement, and the example is so simple it probably runs anyway, but please see the other comments.

**Comment by Outcue** [comment]
Yes, it did run, which made me think I fixed it. I'll make the correct changes.

**Pull request merged**